lib/template.html.erb

```diff
@@ -54,10 +54,9 @@
   <body>
     <h1><%= name %></h1>
```

Comment:
Name is already rendered here, and it would be again if you loop through all PERSONAL_DATA_METHODS